README.md

```diff
@@ -218,20 +218,19 @@ There are 2 common problems:
 * The required codec library is not installed (vpx in this example). In this case, please refer to the [available codecs](#available-codecs).
 * Pkg-config fails to find the `.pc` files for this codec ([reference](https://people.freedesktop.org/~dbn/pkg-config-guide.html#using)). In this case, you need to find where the codec library's `.pc` is stored, and let pkg-config knows with: `export PKG_CONFIG_PATH=/path/to/directory`.
 
+### Roadmap
+The library is used as a part of our WebRTC implementation. Please refer to that [roadmap](https://github.com/pion/webrtc/issues/9) to track our major milestones.
```

Comment:
I believe MediaDevices API is not a *part* of WebRTC but can be used with. 
- Web APIs
  - WebRTC https://developer.mozilla.org/en-US/docs/Web/API/WebRTC_API
  - Media Capture and Streams API https://developer.mozilla.org/en-US/docs/Web/API/Media_Capture_and_Streams_API
  - ...

Reply:
Agreed :) I changed the wording :)